rules/directory-build-props/rule.md

```diff
@@ -0,0 +1,111 @@
+---
+seoDescription: Do you use Directory.Build.Props to simplify multi-project build configurations?
```

Comment:
Hi @danielmackay 

- [ ] Can you use ChatGPT to create a description that summarizes the content? Repeating the tilte is no good for SEO